build-data-capturing-gradle-samples/capture-maxparallelforks/gradle-maxparallelforks.gradle

```diff
@@ -0,0 +1,10 @@
+def buildScan = project.extensions.findByName('buildScan')
```

Comment:
Did you verify these samples are configuration cache compatible?

Reply:
Yep they are both configuration cache compatible. Updated the description with build scans showing the configuration cache being reused.